fulfiller.go

```diff
@@ -0,0 +1,12 @@
+package capnp
+
+// A Fulfiller supplies a value for a pending promise.
```

Comment:
Nit:  the name `Fulfiller` captures half of this type's functionality.  Ideally, we'd have a name that also captures the "reject" half.   

How about `Resolver`?

Reply:
Done